server/entities/playlist.js

```diff
@@ -0,0 +1,10 @@
+class PlayList{
+    constructor(playlistItem){
+        this.id = playlistItem.id;
+        this.name = playlistItem.name;
+        this.ownerId = playlistItem.owner.id;
+        this.isPublic = playlistItem.public;
+        this.trackCount = playlistItem.tracks.total;
+    }
+}
+module.exports = PlayList;
```

Comment:
I have created "Entities" folder because "Models" contains mongoose model and it is a different one. But we can put into "Models", if you want.